test/fixtures/mson/enum-sample.json

```diff
@@ -197,6 +223,26 @@
                             "content": [
                               {
                                 "element": "number"
+                              },
+                              {
+                                "element": "number",
+                                "content": 3
+                              },
+                              {
+                                "element": "number",
+                                "content": 4
+                              },
+                              {
+                                "element": "number",
+                                "content": 5
+                              },
+                              {
+                                "element": "number",
+                                "content": 6
+                              },
+                              {
+                                "element": "number",
+                                "content": 1
```

Comment:
The `e7` in this file is wrong. https://github.com/apiaryio/drafter/pull/532/files#diff-882a86245349acaecd3ba383d5503251R350

Reply:
Due to there is no change against older code let make separated issue.